instance/templates/instance/ansible/swift.yml

```diff
@@ -0,0 +1,6 @@
+EDXAPP_SWIFT_AUTH_VERSION: '2'
+EDXAPP_SWIFT_USERNAME: '{{ user }}'
+EDXAPP_SWIFT_KEY: '{{ password }}'
+EDXAPP_SWIFT_TENANT_NAME: '{{ tenant }}'
+EDXAPP_SWIFT_AUTH_URL: '{{ auth_url }}'
+EDXAPP_SWIFT_REGION_NAME: '{{ region }}'
```

Comment:
@omarkan Would it be possible to implement logic similar to https://github.com/edx/edx-platform/pull/11286/files#diff-8126d0664ba3411882f0ceac7d41e2e5R22 in edx/configuration#2723?  This would allow us not to set explicitly set `EDXAPP_DEFAULT_FILE_STORAGE`, so the code would work for edx-platform versions that do include Swift support as well as those that don't.